webapp/src/js/controllers/reports-content.js

```diff
@@ -89,7 +89,16 @@ var _ = require('underscore');
               $scope.deselectReport(change.doc);
             });
           } else {
-            $scope.refreshReportSilently(change.doc);
+            var selected = $scope.selected;
+            $scope.refreshReportSilently(change.doc)
+              .then(function() {
+                if(selected[0].formatted.verified !== change.doc.verified) {
+                  $scope.selected = selected;
+                  $timeout(function() {
+                    $scope.selected[0].formatted.verified = change.doc.verified;
+                  });
+                }
+              });
```

Comment:
Why is this block necessary? I feel like the `$scope.refreshReportSilently(change.doc);` bit should update the selected report with the appropriate verification status. As in, it should just work without this change.

Reply:
It does update the model but it does not refresh the ui.